### DISCUSSION TITLE
Export invoices .xml and VAT prefix

### DISCUSSION THREAD

**Discussion by ciucaadrian11** [Intermesh/groupoffice#989] [General]
Hello Group Office,
Begining with this week I started to test/use this beatifull software and I can say that until now I am very satisfied about its functionalities. Now I am testing this software using the license offered free for two month and I am prety sure that I will continue to buy the license after this period.

I would like to know the follwing information:

- Can I export the invoices, orders etc. in format .xml? I am from Romania and here over 80% of accountants use the SAGA program for accounting. Also, in format .xml the accountants can upload the invoices, orders in any invoicing, accounting program.

- How I can remove the Prefix from VAT (RO XXXXX). In Romania the VAT is named CUI or CIF and can be with RO prefix or without RO. I set the VAT XXXX for a customer and when I made an invoice for this custumer, the software automatically adds the RO prefix...

- Begining with 2022, in Romania was implemented e-invoice (e-factura). That mean, all invoices must go through the government finance system (these can be loaded manually or sent automatically if the software is linked (API) to the ANAF (National Agency for Fiscal Administration). The format of invoices that should send to the ANAF I think should be XML (UBL 2.1).

However, I saw that many other countries have implemented the e-invoice system. (for more information please consult de follwing link:
This is the link with the specification from finance ministery regarding to the "How to be the invoices, format invoices, etc.: https://mfinante.gov.ro/web/efactura/informatii-tehnice
Other information:
https://www.fonoa.com/countries/romania/e-invoicing-and-digital-reporting
https://snitechnology.net/e-invoicing/

Thank you in advance fpr your support!

**Comment by mschering** [comment]
We have no support for this yet. If you're interested we could implement the XML format you desire for you. Please contact us at info@intermesh.nl if you want us to create this feature for you.

**Reply by ciucaadrian11** [reply]
Hello,
I open a ticket from my account at the following link: https://www.group-office.com/account#support
It is necessary to send also an email at the address: info@intermesh.nl?

And related to Prefix of VAT... It is the any possibility to make any change to don't appear automatically the prefix like RO, DE, etc...

**Reply by ciucaadrian11** [reply]
I forget to mention that I have installed this software through softaculous, direct from cPanel on my domain. I am currently using version 6.6.159. because this is the latest version available on softaculous, How can I update to 6.7?
Thank you!

**Reply by mschering** [reply]
Softaculous is working on a 6.7 version. They are always a bit later with releasing versions.
